monitor/src/main/webapp/WEB-INF/views/home.jsp

```diff
@@ -3,20 +3,50 @@
 <%@ taglib prefix="sec"
 	uri="http://www.springframework.org/security/tags"%>
 <%@ taglib uri="http://sargue.net/jsptags/time" prefix="time"%>
-
+<c:url value="/reload"  var ="reloadUrl"/>
+<script>
+	window.setInterval(function() {
+		$.ajax({
+			type : "GET",
+			dataType : "json",
+			url : "${reloadUrl}",
+			success : function(data) {
+				$('#ajax_reload_alert').hide()
+				data.forEach(update)
+			},
+			error : function(xhr, status, error) {
+				console.error(xhr.responseText)
+				console.error(status)
+				console.error(error.message)
+				$('#ajax_reload_alert').show()
+			}
+		});
+	}, 60000);
+	function update(data) {
+		$('#name_' + data.id).text(data.name)
+		$('#desc_' + data.id).text(data.description)
+		$('#url_' + data.id).text(data.healthUrl)
+		$('#status_' + data.id).text(
+				"App status is: " + data.lastAppTest.status)
+	}
```

Comment:
the only thing that in most cases will change is the time the last check was run, which isn't updated here.